AGENTS.md

```
@@ -0,0 +1,116 @@
+# AGENTS.md
+
+## 项目概述
+
+NestJS + React 全栈 Monorepo 学习项目。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)